blog/all.rkt

```diff
@@ -9,6 +10,13 @@
         #:url (rewrite-for-testing "https://blog.racket-lang.org/")))
 (register-identity blog-site)
 
+(define-runtime-path blog-dir ".")
+(define css-dir (simplify-path (build-path blog-dir "css/")))
+(define raco-path (let-values ([(base name _) (split-path (find-exe))])
+                    (build-path base "raco")))
+(system (format "~a frog -b" raco-path)) ; frog rebuild: generates blog html
+(system (format "~a pollen render ~a" raco-path css-dir)) ; pollen rebuild: generates css
+
```

Comment:
It is probably wise to use `system*` here. And if this might run on a windows machine, use `raco.exe` (but maybe that doesn't matter).